samples/textDetection.js

```diff
@@ -89,7 +88,7 @@ Index.prototype.lookup = function(words, callback) {
       self.tokenClient.smembers(word, cb);
     };
   });
-  async.parallel(tasks, callback);
+  Promise.all(tasks).then(callback);
```

Comment:
Same comment here, I think you can just `return Promise.all(taskss);`
and line 83 make into
`Index.prototype.lookup = async words => {`

Reply:
Done! https://github.com/vijay-qlogic/nodejs-vision/pull/1/commits/bb6ccacc6481592a693e8d7c357219f88de87c96